lib/srv/reexec.go

```diff
@@ -44,6 +44,18 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type FileFD = uintptr
+
+const (
+	CommandFile FileFD = 3 + iota
+	ContinueFile
+	X11File
+	PTYFile
+	TTYFile
+
+	ExtraFD FileFD = 6
```

Comment:
Do you intent for `ExtraFD` and `PTYFile` to both have the value 6?

Reply:
Yes I did, because a process with a terminal shouldn't get passed additional extra files (was my thought process). Processes without a terminal are probably Teleport re-execs, so not allowing extra files for processes with a terminal should be fine imo. I should add checks for a terminal when opening the extra files though :+1: